docs/get-started/try-viam/rover-resources/rover-tutorial-2.md

```diff
@@ -0,0 +1,383 @@
+
+Mounted above the switch is a low voltage cutoff circuit that can be set to turn off power to the rover when the input voltage drops below a pre-set threshold.
+This can be helpful for preventing batteries fully discharging which can damage lithium ion batteries.
+
+### Battery holders
+
+The Viam Rover 2 comes with two battery holder options.
+The rover nominally operates with four 18650 batteries, but a higher capacity RC-type battery can be mounted into the rear of the rover.
+
+{{% alert title="Note" color="note" %}}
+With either battery option, you must purchase a charger separately.
+{{% /alert %}}
+
+#### 18650 battery pack
+
+{{<imgproc src="get-started/try-viam/rover-resources/viam-rover-2/battery-pack.png" resize="400x" declaredimensions=true alt="A battery pack" >}}
+
+18650 batteries are the nominal power supply recommended for use with the Viam Rover 2.
+An 18650 battery is a lithium-ion rechargeable battery.
+We recommend the button-top type, though either button or flat top can work.
+Any brand is suitable as long as you comply with the battery safety requirements.
+
+#### Mount for RC-type battery
+
+{{<imgproc src="get-started/try-viam/rover-resources/viam-rover-2/RC-mount.png" resize="400x" declaredimensions=true alt="RC battery mount" >}}
+
+You can mount a larger capacity RC-type battery into the rover.
+You must wire the appropriate connecter into the switch circuit.
+
+RC-batteries are lithium-ion rechargeable batteries.
+Caution should always be taken when using such batteries, always comply with the battery safety requirements.
+Check the [safety](#safety) section for more information.
+
+## Safety
+
+Read all instructions fully before using this product.
+
+This product is not a toy and is not suitable for children under 12.
+
+Switch the rover off when not in use.
+
+{{< alert title="Warning" color="warning" >}}
+Lithium-ion batteries may pose a flammable hazard.
+This product requires four 18650 lithium-ion batteries OR an RC-type battery.
+DO NOT connect multiple power sources simultaneously.
+Refer to the battery manufacturer’s operating instructions to ensure safe operation of the Viam Rover.
+Dispose of lithium-ion batteries per manufacturer instructions.
+{{< /alert >}}
+
+{{< alert title="Caution" color="caution" >}}
+Damage may occur to the Raspberry Pi and/or Viam Rover if wired incorrectly.
+Refer to the manufacturer’s instructions for correct wiring.
+{{< /alert >}}
+
+Disclaimer: This product is preliminary and experimental in nature, and is provided "AS IS" without any representation or warranty of any kind.
+Viam does not make any promise or warranty that the product will meet your requirements or be error free.
+Some states do not allow the exclusion or disclaimer of implied warranties, so the above exclusions may not apply to you.
+
+## Setup
+
+### Install Raspberry Pi OS
+
+{{% alert title="Tip" color="tip" %}}
+If you are using another board, you can skip this step.
```

Comment:
Shouldn't we direct them to follow the setup instructions for their board?

Reply:
They still need to complete the next steps. The mounting/wiring part is the main difference

Reply:
I mean the nord setup steps for their board? Like https://docs.viam.com/get-started/installation/prepare/jetson-nano-setup/?